devops/repos/common/init.sh

```diff
@@ -1,7 +1,10 @@
 #!/bin/bash
 
-HIPSYCL_REPO_STAGE_DIR=${HIPSYCL_REPO_STAGE_DIR:-./stage}
+REPO_STAGE_DIR=${REPO_STAGE_DIR:-./stage}
```

Comment:
Can you comment on why this variable needed to be renamed? To be it seems good practice to have our environment variables namespaced using the `HIPSYCL_` prefix.

Reply:
I renamed it because this folder might contain the base and the rocm packages as well. Therefore I thought naming it `HIPSYCL_` would be misleading in a scenario where the rocm and base packages are put in a different stage directory in order to carry out further testing on them. However